README.md

```diff
@@ -14,28 +14,57 @@ Install the package via Composer:
 composer require studiometa/wp-toolkit
 ```
 
+**Requirements**
+
+- PHP >=7.3
+
 ## Usage
 
 ```php
-use Studiometa\WPToolkit\Assets;
-use Studiometa\WPToolkit\Cleanup;
-use Studiometa\WPToolkit\Builders\PostTypeBuilder;
 
 // Load assets from `assets.yaml` configuration
+use Studiometa\WPToolkit\Assets;
 new Assets( get_template_directory() );
 
 // Clean WordPress
+use Studiometa\WPToolkit\Cleanup;
 new Cleanup();
 
 // Create Custom Post Type
+use Studiometa\WPToolkit\Builders\PostTypeBuilder;
+
 $cpt = new PostTypeBuilder( 'product' );
 $cpt->set_labels( 'Product', 'Products' )
   ->set_has_archive( true )
   ->register();
 
 // Create Custom Taxonomy
+use Studiometa\WPToolkit\Builders\TaxonomyBuilder;
+
 $tax = new TaxonomyBuilder( 'product-cat' );
 $tax->set_post_types( 'product' )
   ->set_labels( 'Product Category', 'Product Categories' )
   ->register();
+
+// Create a manager
+use Studiometa\WPToolkit\Managers\ManagerInterface;
+
+class CustomManager implements ManagerInterface {
+  run() {
+    add_action( 'init', array( $this, 'some_action' ) );
+  }
+
+  some_action() {
+    // do something on init
+  }
+}
+
+// Init all managers
+use Studiometa\WPToolkit\Managers\ManagerFactory;
+
+ManagerFactory::init(
```

Comment:
I think we should use the Factory this way :
```php
$factory = new ManagerFactory();
$factory->create(FooManager::class); // Store manager in private class attribute `managers`
$factory->init(); // Run all stored managers
```

Reply:
What would be the benefits over the `init()` static method?

Reply:
I don't really remember why I'd post this comment but I think this is not the static method who bothered me, it's the meaning of "Factory" who disturbed me. The current implementation is not a Factory, it's a facade/helper to run managers.
Maybe we should just loop over managers and run each one (Is it really usefull to have this layer?), otherwise just changing the classname to `ManagerHelper`?

```php
ManagerFactory::init(
  array(
    new CustomManager(),
    new CustomManager1(),
  )
);
// vs.
$managers = array(new CustomManager(), new CustomManager1());
foreach($managers as $manager) {
    $manager->run();
}
```

Reply:
+1 @lusimeon. This isn't a really needed abstraction, it hide a simple logic into the repo which is suppose to be a dependancy in a WP env